resources/qml/Preferences/Materials/MaterialsBrandSection.qml

```diff
@@ -13,7 +13,8 @@ import Cura 1.0 as Cura
 Item
 {
     id: brand_section
-    
+
+    property var text: ""
```

Comment:
We don't need this property. We can get it just as well from the `sectionName` custom property.